submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/data/SubmissionRequestDocDataRepository.kt

```diff
@@ -241,7 +240,7 @@ class SubmissionRequestDocDataRepository(
 
     @Suppress("SpreadOperator")
     private fun createQuery(filter: SubmissionListFilter): Criteria =
-        where("$RQT_PROCESS.$SUB.$SUB_OWNER")
+        where(RQT_OWNER)
```

Comment:
can we havea test for this. We can insert using persistence classes submission request in diferent stages and validate they are (and not) viisible when api call. 

Not a unit test, not a integration test but IMO best we can do